sigopt/objects.py

```diff
@@ -137,13 +136,13 @@ def dumps(obj, indent_level=0):
     indent = ' ' * indent_level
 
     if isinstance(obj, BaseApiObject):
-      return six.u('{0}{1}'.format(indent, str(obj).replace('\n', '\n{0}'.format(indent))))
+      return '{0}{1}'.format(indent, str(obj).replace('\n', '\n{0}'.format(indent)))
     if is_mapping(obj):
       if obj:
-        return six.u('{0}{\n{1},\n{0}}').format(
+        return '{0}{{\n{1},\n{0}}}'.format(
```

Comment:
confirming, this is escaping the `{` so it renders literally, right?

Reply:
yes, not sure how this worked before (maybe .format works slightly differently than f-string) but the tests for this behavior are passing

Reply:
works for me, thanks for checking

Reply:
```
>>> sigopt.objects.ApiObject.dumps({"hello": "there"})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/taylor/sigopt-python/sigopt/objects.py", line 143, in dumps
    return six.u('{0}{\n{1},\n{0}}').format(
ValueError: unexpected '{' in field name
```
it was always broken